R/get_stats_utils.R

```diff
@@ -116,4 +116,4 @@ getEti = function(v, mass) {
 getHdi = function(v, mass) {
   r = HDInterval::hdi(v, credMass = mass)
   d = data.table(lower = r[1L], upper = r[2L])
-  return(d)}
+  return(d)}
```

Comment:
Add back the line return.

Reply:
See above. I'm not sure what that's all about.